### DISCUSSION TITLE
Preconfigured animation presets?

### DISCUSSION THREAD

**Discussion by saif-o99** [nandorojo/moti#310] [General]
I just came from react-native-animatable, found this library and its amazing. Thanks for the hard work.

is there any chance we gonna a preconfigured animation presets just like react-native-animatable? it makes it pretty easy to use this lib.

eg:
- fadeIn
- fadeInDown
- fadeInDownBig
- fadeInUp
- fadeInUpBig
- fadeInLeft
- fadeInLeftBig
- fadeInRight
- fadeInRightBig

and i saw many people liked the idea : https://github.com/oblador/react-native-animatable/issues/389#issuecomment-1023034524

**Comment by nandorojo** [comment]
i’ve always preferred to keep these abstractions in user land. you could create a wrapper component around the moti view (i recommend doing this anyway) that takes a string prop and maps that string to an animate prop

**Reply by saif-o99** [reply]
We can keep the abstraction ofcourse and add these presets. but not problem if you see this out of the lib scope
